samples/dotnetcore/Startup.cs

```diff
@@ -0,0 +1,140 @@
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddAuthentication(options =>
+            {
+                options.DefaultAuthenticateScheme = OpenIdConnectDefaults.AuthenticationScheme;
+                options.DefaultSignInScheme = OpenIdConnectDefaults.AuthenticationScheme;
+                options.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
+                options.DefaultScheme = OpenIdConnectDefaults.AuthenticationScheme;
+            })
+            .AddCookie(OpenIdConnectDefaults.AuthenticationScheme, options =>
+            {
+                options.Cookie.SameSite = SameSiteMode.None;
+            })
+            .AddOpenIdConnect("oidc", options =>
+            {
+                options.Authority = "https://api.mpin.io";
+                options.ClientId = this.Configuration["CLIENT_ID"];
+                options.ClientSecret = this.Configuration["CLIENT_SECRET"];
+                options.ResponseType = OpenIdConnectResponseType.Code;
+                options.AuthenticationMethod = OpenIdConnectRedirectBehavior.RedirectGet;
+                options.Scope.Clear();
+                options.Scope.Add("openid email profile");
+                options.CallbackPath = this.GetCallbackPath();
+
+                options.GetClaimsFromUserInfoEndpoint = true;
+                options.SaveTokens = true;
+
+                options.ClaimsIssuer = "oidc";
+                options.ProtocolValidator.RequireNonce = false;
+
+                options.ClaimActions.MapJsonKey("email_verified", "email_verified", "bool");
+                options.ClaimActions.MapJsonKey("sub", "sub", "string");
+                options.Events = new OpenIdConnectEvents()
+                {
+                    OnAuthenticationFailed = c =>
+                    {
+                        c.HandleResponse();
+                        c.Response.StatusCode = StatusCodes.Status400BadRequest;
+                        return c.Response.WriteAsync("Authentication Failed");
+                    },
+                };
+            });
+
+            // Add framework services.
+            services.AddControllersWithViews();
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseAuthentication();
+            app.UseAuthorization();
+            app.UseRouting();
+
+            app.Run(async context =>
+            {
+                var response = context.Response;
+
+                if (context.Request.Path.Equals("/signout"))
```

Comment:
I thought that we decided there should be no logout/signout functionality in the samples, since there is no such thing in OIDC or am I mistaken?
Again - whatever we decide, we should keep it consistent within all samples and add that endpoint there.

Reply:
I kept that for testing purposes, but you're right.. I will remove that